.travis.yml

```diff
@@ -20,9 +20,7 @@ deploy:
   provider: script
   script: bash docker_push
   on:
-    branches:
-      only:
-      - master
-      - release
+    all_branches: true
+    condition: $TRAVIS_BRANCH =~ ^master|release$
```

Comment:
wait i'm actually confused, doesn't that contradict [this](https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches), i'm gonna ping you offline